Compilation fails on kernel 4.19 (caused by the new security_bpf_map event)

While compiling, I get the following message, which seems to be related to the new security_bpf_map event (this does not happen on the same environment with kernel 5.9):

```
	-O2 -emit-llvm -c -g tracee/tracee.bpf.c -o dist/tracee.bpf.4_19_192-1-MANJARO.v0_5_2-4-g7feb572.ll
tracee/tracee.bpf.c:3318:46: error: incomplete definition of type 'struct bpf_map'
    save_to_submit_buf(submit_p, (void *)&map->id, sizeof(int), UINT_T, DEC_ARG(0, *tags));
                                          ~~~^
/lib/modules/4.19.192-1-MANJARO/build/include/linux/bpf-cgroup.h:15:8: note: forward declaration of 'struct bpf_map'
struct bpf_map;
       ^
tracee/tracee.bpf.c:3320:43: error: incomplete definition of type 'struct bpf_map'
    save_str_to_buf(submit_p, (void *)&map->name, DEC_ARG(1, *tags));
                                       ~~~^
/lib/modules/4.19.192-1-MANJARO/build/include/linux/bpf-cgroup.h:15:8: note: forward declaration of 'struct bpf_map'
struct bpf_map;
       ^
2 errors generated.
make: *** [Makefile:77: dist/tracee.bpf.4_19_192-1-MANJARO.v0_5_2-4-g7feb572.o] Error 1
```

--- a/tracee-ebpf/tracee/tracee.bpf.c
+++ b/tracee-ebpf/tracee/tracee.bpf.c
@@ -40,6 +40,7 @@
 #include <linux/ipv6.h>
 
 #include <uapi/linux/bpf.h>
+#include <linux/bpf.h>
 #include <linux/kconfig.h>
 #include <linux/version.h>
 

